### PR TITLE
Fixes #7902: Add support to cfengine bootstrap in server

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/7902

Replacing previous PR: https://github.com/Normation/rudder-packages/pull/882

### CODE REVIEW

**Comment by peckpeck** [comment]
@jooooooon could you validate this PR without merging it so that I can our new merge workflow

**Comment by peckpeck** [comment]
+test

File: rudder-webapp/SOURCES/rudder-upgrade [hunk 1]
```diff
@@ -58,6 +58,7 @@ trap anomaly_handler ERR INT TERM
 # - 3.0.0        : Replace varchar by text in the database
 # - 3.0.2        : Check for the PostgreSQL version (>= 8.4)
 # - 3.0.14       : Add index on nodeconfigids for expectedreportsnodes
+# - 3.0.16       : Add masterfiles in the server
```
Comment (jooooooon) [inline rudder-webapp/SOURCES/rudder-upgrade]: Since this targets 3.1, this should be 3.1.10

File: rudder-webapp/SOURCES/rudder-upgrade [hunk 2]
```diff
@@ -355,6 +356,17 @@ upgrade_ncf() {
 
 }
 
+# Upgrade masterfiles
+upgrade_masterfiles() {
+  # - 3.0.16  Now we should have a masterfiles initialized from initial promises
```
Comment (jooooooon) [inline rudder-webapp/SOURCES/rudder-upgrade]: Same here for version

**Comment by jooooooon** [comment]
Aside from the version issue, this looks good to me.

**Comment by peckpeck** [comment]
Commit modified

**Comment by jooooooon** [comment]
:+1:

**Pull request merged**